apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -833,6 +866,7 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     function _isAccepted(Vote storage _vote, Setting storage _setting) internal view returns (bool) {
         uint256 yeas = _vote.yea;
         uint256 nays = _vote.nay;
+        // Note: minor storage optimization might be to pull the support and quorum variables out first
```

Comment:
The reason for this is that both of these vars are uint64s and should be accessible via one SLOAD (we should make sure to order them as such!). However, since they're used like this, I'm not sure what solidity will do.

Reply:
Addressed in eff6dec49286034c16de77d3bd4f40c67d8d56ee